src/Zforce.cpp

```diff
@@ -73,13 +79,13 @@ int Zforce::Read(uint8_t * payload)
 #if USE_I2C_LIB == 1
   int status = 0;
 
-  status = I2c.read(ZFORCE_I2C_ADDRESS, 2);
+  status = I2c.read(this->i2cAddress, 2);
 
   // Read the 2 I2C header bytes.
   payload[0] = I2c.receive();
   payload[1] = I2c.receive();
 
-  status = I2c.read(ZFORCE_I2C_ADDRESS, payload[1], &payload[2]);
+  status = I2c.read(this->i2cAddress, payload[1], &payload[2]);
 
   return status; // return 0 if success, otherwise error code according to Atmel Data Sheet
 #else
```

Comment:
What about the ZFORCE_I2C_ADDRESS in the else-block? I.e. when using the Wire library.

Reply:
Great =)